I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 5, in script
    define config.say_speed = 0.05  # Adjust this to change the typing speed (lower is faster)
  File "renpy/common/000namespaces.rpy", line 33, in set
    setattr(self.nso, name, value)
Exception: config.say_speed is not a known configuration variable.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 5, in script
    define config.say_speed = 0.05  # Adjust this to change the typing speed (lower is faster)
  File "C:\Users\Carlijn\Documents\renpy-8.1.3-sdk\renpy\ast.py", line 2508, in execute
    self.set()
  File "C:\Users\Carlijn\Documents\renpy-8.1.3-sdk\renpy\ast.py", line 2526, in set
    ns.set(self.varname, value)
  File "renpy/common/000namespaces.rpy", line 33, in set
    setattr(self.nso, name, value)
  File "C:\Users\Carlijn\Documents\renpy-8.1.3-sdk\renpy\defaultstore.py", line 101, in __setattr__
    raise Exception('config.%s is not a known configuration variable.' % (name))
Exception: config.say_speed is not a known configuration variable.

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
Code: Uroburos Alpha 1
Thu Mar 20 17:59:01 2025
